### ISSUE TITLE
Google Analytics Support

### ISSUE THREAD

**Issue by adammhaile** [avianto/hugo-kiera#23]
I had to add the following to the `header.html` partial:

```
{{ template "_internal/google_analytics_async.html" . }}
```

Once I did that everything worked. Looks like it was just left out from the base template?

**Comment by donnyv12** [comment]
yeah, I had the same experience and used the above. @adammhaile you should put in a PR with the fix (or I can if you want).

**Comment by funkydan2** [comment]
There's already a [PR](https://github.com/avianto/hugo-kiera/pull/24) for this. It's been there for a while. Feel free to use my fork if that helps.
